templates/re-base/scripts/install.js

```diff
@@ -16,5 +16,5 @@ files.forEach(function(name)
   const path = `node_modules/react-scripts/config/${name}.js`
   const data = readFileSync(path, 'utf8')
 
-  writeFileSync(path, data.replace('babelrc: false', 'babelrc: true'))
+  writeFileSync(path, data.replace('babelrc: true,', 'babelrc: true,\nplugins: [\'react-native-web\'],'))
```

Comment:
The file originally has `babelrc: false`, if you change it that way then it would not work because it would not be find. Also, if you are doing this more deep changes, probably it would be better to parse the JSON content and change it as an object, just like it was done with the `.babelrc` file before.

Reply:
Regarding `.babelrc` file support, although we don't need to enable it anymore... would make it sense to enable it ourselves, or should we left it disable as `react-scripts` does by default?

Reply:
I think we should leave it disabled since it is the default.

Reply:
I'm fine with that. Then in that case, we should only add the `plugins` field, but being Javascript files they are not so easy to manipulate as JSON ones... It would need a refactor, but if you are not into the mood, it's ok for me.

Reply:
Yeah, I'm not thrilled with it but since this is hacky anyway, I didn't put too much thought into it.